app/src/main/java/com/hedvig/app/feature/offer/ui/composable/VariantButton.kt

```diff
@@ -0,0 +1,90 @@
+package com.hedvig.app.feature.offer.ui.composable
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Card
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+import com.hedvig.app.ui.compose.theme.hedvigBlack
+import com.hedvig.app.ui.compose.theme.hedvigBlack12percent
+
+@Composable
+fun VariantButton(
```

Comment:
Looking at the preview, the radio button seems to be a bit too high maybe? Is there a FIgma file for this, I couldn't find one. Maybe we can fix it up a bit before merging if my eyes are not deceiving me.